client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -409,22 +409,24 @@ const Editor = (props: EditorProps): JSX.Element => {
       return accessibility;
     };
 
-    const setTabTrapped = (trapped: boolean) => {
+    const setTabTrapped = (trapped: boolean, announce = true) => {
```

Comment:
```suggestion
    const setTabTrapped = (
      trapped: boolean,
      opts: { announce: boolean } = { announce: true }
    ) => {
```
Nitpick but `setTabTrapped(false, false)` doesn't make sense to me without the context.